compiler-plugin-test/src/test/kotlin/it/unibo/collektive/test/TestExplicitAlign.kt

```diff
@@ -0,0 +1,29 @@
+package it.unibo.collektive.test
+
+import io.kotest.core.spec.style.FreeSpec
+import it.unibo.collektive.test.util.CompileUtils
+import it.unibo.collektive.test.util.CompileUtils.ProgramTemplates
+import it.unibo.collektive.test.util.CompileUtils.warning
+import org.jetbrains.kotlin.compiler.plugin.ExperimentalCompilerApi
+
+@OptIn(ExperimentalCompilerApi::class)
+class TestExplicitAlign : FreeSpec({
+    val testingProgramTemplate = CompileUtils.testingProgramFromTemplate(ProgramTemplates.SINGLE_AGGREGATE_LINE)
+
+    "An explicit align function should not be used" - {
```

Comment:
Please, rewrite the test names to be more "flowable" when you read it (nested test assertions).

For instance, I would change as follows:

```kotlin
"The `align` function" - {
  "should produce a warning when used explicitly" {
  ...
}
```

Similarly for the rest of the test assertions.